runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -277,76 +277,47 @@ ClientSessionData::printStats()
    j9tty_printf(PORTLIB, "\tTotal size of cached ROM classes + methods: %d bytes\n", total);
    }
 
+ClientSessionData::ClassInfo::ClassInfo() :
+   _romClass(NULL),
```

Comment:
I counted 30 fields in the hpp file, but obly 29 are initialized. One is missing as far as I can tell

Reply:
_interfaces(NULL) was missing